cmd/utils/flags.go

```diff
@@ -307,6 +307,11 @@ var (
 		Usage: "Number of trie node generations to keep in memory",
 		Value: int(state.MaxTrieCacheGen),
 	}
+	DatabaseHandles = cli.IntFlag {
```

Comment:
travis lint error found at this line. simply remove space, `IntFlag {` to `IntFlag{` will work.